### ISSUE TITLE
GraphQL: sitemap pagination does not work

### ISSUE THREAD

**Issue by d--j** [nystudio107/craft-seomatic#1492]
### Describe the bug

The GraphQL sitemap resolver needs to also understand the new file names (optional `-p\d+` suffix). Currently it does not and thus will only ever get the first page:

https://github.com/nystudio107/craft-seomatic/blob/fad047b2de8ff1da6b0890aba70799e9277e9408/src/gql/resolvers/SitemapResolver.php#L146

https://github.com/nystudio107/craft-seomatic/blob/fad047b2de8ff1da6b0890aba70799e9277e9408/src/models/SitemapTemplate.php#L109

### Expected behaviour

If you pass in a GrapQL query like

```graphql
query GetSitemapXml {
  seomatic {
    sitemaps(site: "site_handle", filename: "sitemaps-1-section-sectionProjects-1-sitemap-p2.xml") {
      filename
      contents
    }
  }
}

```

you should get back the requested page.

### Versions
- Plugin version: 4.1.0
- Craft version: Craft Pro 4.10.5

**Comment by khalwat** [comment]
Will fix, thanks

**Comment by khalwat** [comment]
Addressed. 

Craft CMS 3:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop as 3.5.1”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 4:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop-v4 as 4.1.1”,
```
Then do a `composer clear-cache && composer update`

…..

Craft CMS 5:

You can try it now by setting your semver in your `composer.json` to look like this:
```json
    "nystudio107/craft-seomatic": "dev-develop-v5 as 5.1.1”,
```
Then do a `composer clear-cache && composer update`

**Comment by khalwat** [comment]
Released as SEOmatic `3.5.1`, `4.1.1`, and `5.1.1` for Craft 3, 4, & 5 respectively
